qa/automation/CareerPortal.Tests.Scenarios/1.5 Details of position.feature

```diff
@@ -0,0 +1,123 @@
+Feature: 1.5 Details of position
+	Checking details of position and contact form
+
+Background:
+Given There is one position:
+		| title            | Senior QA             |
+		| description      | custom text           |
+		| location         | location text         |
+		| responsibilities | responsibilities text |
+		| what_we_expect   | what we expect text   |
+		| nice_to_have     | nice to have text     |
+		| we_offer         | we offer text         |
+
+Scenario: 1.5_01 Position contains all the details
+Given Opened '/positions' page
+When User clicks on position 'Senior QA'
+Then There are details about the position described in the background section
+
+Scenario: 1.5_02 'Send CV' button on the top
+Given Opened position 'Senior QA' page
+When User clicks 'Send CV' button
+Then Page is scrolled to the form where user can enter Name/Email etc
+
+Scenario: 1.5_03 Sending CV - positive test
+Given Opened position 'Senior QA' page
+When User enters name 'Pavel'
```

Comment:
you will not be able to send a resume, because the name must be in the format <Name><Space><Last Name>. If one word is written in this field without a space, an error will appear and the resume will not be sent. I have the same error in my code, relax)

Reply:
Thank you for this remark!
Yes, I made typo. In Scenarios 05 and 06 I checked it.